lib/src/components/NxTableBody/NxTableBody.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {Props} from './types';
+export {Props} from './types';
+
+const NxTableBody = function NxTableBody(props: Props) {
+  const {className, ...attrs} = props;
+
+  return (
+    <tbody className={className} {...attrs} />
```

Comment:
If you aren't setting any additional classnames there's no need to pull out the `className` prop.  In fact, I'd argue there's no reason to have this component

Reply:
I guess the only argument I have for keeping this component is consistency. Without this component you end up with stuff like this.

```
<NxTable>
  <NxTableHead>
    ...
  </NxTableHead>
  <tbody>
    ...
  </tbody>
</NxTable>
```

I'll go ahead and remove the `NxTableBody` component.

Reply:
> I'll go ahead and remove the NxTableBody component.

I mean, consistency is potentially a valid argument

Reply:
> > I'll go ahead and remove the NxTableBody component.
>
> I mean, consistency is potentially a valid argument

I really did see the value of the consistency, maybe even liked it better that way.  What do others think? @sonatype/iq-laurel

Reply:
It's easy enough to bring back if we think it's better.

Reply:
> I really did see the value of the consistency, maybe even liked it better that way. What do others think? @sonatype/iq-laurel

I vote for consistency. The example above without this component looks really weird.

Reply:
I added it back for the loading/error functionality anyway.